_data/meltano/utilities/elementary/elementary.yml

```diff
@@ -0,0 +1,80 @@
+name: elementary
+label: Elementary
+domain_url: https://docs.elementary-data.com/introduction
+namespace: elementary
+pip_url: elementary-data[bigquery]=0.5.3 git+https://github.com/potloc/elementary-ext.git
```

Comment:
@SBurwash this hardcodes the extras for Bigquery. If users of the plugin wanted to use Snowflake instead, they'd need to override `pip_url` in their project:

```diff
- elementary-data[bigquery]=0.5.3 git+https://github.com/potloc/elementary-ext.git
+ elementary-data[snowflake]=0.5.3 git+https://github.com/potloc/elementary-ext.git
```

It's probably not a big deal if that's all users need to change.

cc @pnadolny13 @tayloramurphy

Reply:
@edgarrmondragon I agree we could generalize it. 

The only thing is that my initialize function is based on bigquery because I wanted a product that was functional, but also did not want to reproduce the DBT integration (as this is dependant on the user having an existing DBT integration).

I'm open to any suggestions on how to tackle the fact that this is an extension of DBT, and not an independent plugin

Reply:
@edgarrmondragon what's a reasonable path here? It looks like boht the pip_url and the profiles-dir setting need to be specific to the users adapter. 

Likely we could just update the next steps to explicitly have the user update their pip url and the profiles-dir based on their data warehouse. I don't want this merged with a specific database in place and I don't want multiple entries of Elementary on the Hub. 

Any other option I'm missing?

Reply:
> Likely we could just update the next steps to explicitly have the user update their pip url and the profiles-dir based on their data warehouse.

@tayloramurphy `next_steps` is a good place for that. Setting the `pip_url` to

```
pip_url: elementary-data==0.5.3 git+https://github.com/potloc/elementary-ext.git
```

and instruction users to override it, and also update their dbt profile 👍